applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -224,7 +254,33 @@ def __main():
                         const=True,
                         help="Run nrfjprog sequentially instead of in \
                         parallel.")
+    #DFU relative option
+    parser.add_argument("-m", "--min_b0n", default=False,
+                        action='store_true',
+                        help="net core bootloader use minimal size build")
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("-M", "--mcuboot", default=False,
+                        action='store_true',
+                        help="enable internal flash dfu")
+    group.add_argument("-e", "--external_flash", default=False,
+                        action='store_true',
+                        help="enable external flash dfu")
+
     options = parser.parse_args(args=sys.argv[1:])
+    global USE_MCUBOOT
+    global USE_EXTERNAL_FLASH
+    global USE_MIN_B0N
+    global PWD
+
+    if options.min_b0n:
+        USE_MIN_B0N = options.min_b0n
+    if options.mcuboot:
+        USE_MCUBOOT = options.mcuboot
+    elif options.external_flash:
+        USE_EXTERNAL_FLASH = options.external_flash
+    PWD=os.getcwd()
```

Comment:
Depending on if https://github.com/nrfconnect/sdk-nrf/pull/7542 get merged first, this will no longer be necessary.

Reply:
Sure.
Will refactor after https://github.com/nrfconnect/sdk-nrf/pull/7542 merged.

Reply:
Use NRF5340_AUDIO_FOLDER insteal PWD+ relative path.